assets/js/theme/common/product-details.js

```diff
@@ -401,13 +401,28 @@ export default class ProductDetails extends ProductDetailsBase {
             const $cartQuantity = $('[data-cart-quantity]', modal.$content);
             const $cartCounter = $('.navUser-action .cart-count');
             const quantity = $cartQuantity.data('cartQuantity') || 0;
+            const $promotionBanner = $('[data-promotion-banner]');
+            const $backToShopppingBtn = $('.previewCartCheckout > [data-reveal-close]');
+            const $modalCloseBtn = $('#previewModal > .modal-close');
+            const bannerUpdateHandler = () => {
+                const $productContainer = $('#main-content > .container');
```

Comment:
Do we really need here child selectors? In case if, for example, someday wrapper will be added they may not found an element

Reply:
I believe we do. There are different modal types can be on PDP and these buttons don't have a unique identifiers so we need to be sure this bannerHandler will be invoked through clicking on mentioned buttons in preview modal only.